static/js/publisher/publicise.js

```diff
@@ -39,30 +39,26 @@ function initSnapButtonsPicker() {
 
 const getCardPath = (snapName, options = {}) => {
   const path = `/${snapName}/embedded`;
-  let params = "";
+  let params = [];
```

Comment:
you changed it!? Now I've wasted some shower time thinking about your way!

Reply:
Previous worked fine with 1-2 params, but doesn't scale well.